builder/amazon/common/ami_config.go

```diff
@@ -128,3 +146,23 @@ func (c *AMIConfig) prepareRegions(accessConfig *AccessConfig) (errs []error) {
 	}
 	return errs
 }
+
+// See https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CopyImage.html
+func validateKmsKey(kmsKey string) (valid bool) {
+	kmsKeyIdPattern := `[a-zA-Z1-6-]+$`
```

Comment:
```suggestion
	kmsKeyIdPattern := `[a-zA-Z0-9-]+$`
```

Reply:
Switched to checking for the hexadecimal digits in the UUID instead of what my sleep deprived brain told me was the regex for hex.